unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/CatalogController/CatalogController.cs

```diff
@@ -169,14 +170,18 @@ public static Promise<WearableItem> RequestWearable(string wearableId)
 
         return promiseResult;
     }
-
+    
     public static Promise<WearableItem[]> RequestOwnedWearables(string userId)
     {
         Promise<WearableItem[]> promiseResult;
 
         if (!awaitingWearablesByContextPromises.ContainsKey(OWNED_WEARABLES_CONTEXT))
         {
             promiseResult = new Promise<WearableItem[]>();
+
+            // TODO: find a better solution to avoid executing a request when running tests, mocking the running instance
+            if (EnvironmentSettings.RUNNING_TESTS) return promiseResult;
+            
```

Comment:
This is necessary? I can see that this only request the wearable to the kernel and the kernel shouldn't be active in the unit test so I don't know if this help avoid the issue

Reply:
This line avoids a timeout exception produced by requesting the wearables during tests. This timeout can break any given test that is running at that specific time. I will try to find a workaround by mocking this behaviour so production code is not affected

Reply:
`CatalogController` now has a method `Clear` which resets all the pending promises made by any other test. This is executed on `TearDown`  at `IntegrationTestSuite_Legacy` class which is used by most of the play mode tests